First pass

rogebrd opened the pull request.

rogebrd committed:
First pass

avi-riptide commented on backend/docker-compose.yml:
```diff
@@ -0,0 +1,17 @@
+FROM ubuntu:16.04
+
+MAINTANER Brad Rogers "brad12rogers@gmail.com"
```
you spelled MAINTAINER incorrectly. not sure if that's an issue but it could be

rogebrd committed:
Get main Docker working

rogebrd committed:
Push up progress

rogebrd committed:
[Backend] Update docker compose to pull tokens from .env

rogebrd committed:
[Backend] Add Stock Client for fetching stock prices
- Add token client for fetching tokens needed for stock client

rogebrd committed:
[Backend] Refactor source code to live under src/
- Move app.py and sage.py to live under src/
- Add requests to requirements.txt

rogebrd committed:
[Backend] Update bundle script to reflect new src tree

rogebrd committed:
[Backend] Add login helper and default logging
- Add login manager to handle logins which return tokens
- Add handler for authenticating requests

rogebrd committed:
[Cloud Formation] Update cloud formation template to reflect docker/elb state
- Remove all resources except for dynamodb tables
- Add token table

rogebrd committed:
[Desktop] Add Login Page
- Add in simple navigation
- Remove stockTracker since logic has moved to backend

rogebrd committed:
[Backend] Add /sidebar endpoint

rogebrd committed:
[Backend] Finish sidebar functionality

rogebrd committed:
[Desktop] Update sidebar logic
- Move sidebar to fetch its own content
- Update parser to take in static values rather than types
- Update backend logic to determine if stipend account
